server/map_tree_updater.go

```diff
@@ -110,44 +109,36 @@ func (t *mapTreeUpdater) update(ctx context.Context, tx storage.MapTreeTX, nodes
 }
 
 type txAccessor struct {
-	// hashes is a cache of node hashes that Get returns directly instead of
-	// calling GetMerkleNodes. Used only if preload is true.
-	hashes  map[tree.NodeID2][]byte
-	preload bool
-
-	tx  storage.MapTreeTX
-	rev int64
+	mtu *mapTreeUpdater
+	// read is a cache of tree tiles that Get returns directly instead of calling
+	// GetTiles. It is initialized by the first Get call if not specified.
+	read *smt.TileSet
```

Comment:
read a verb which makes it sound more like a function. 
Could we make it a noun like readCache?

Reply:
Could you make it clearer what happens if the readCache is missing a row? Does that make it a cache miss or does that produce an error or some kind?

Reply:
Renamed to `tiles`, and improved commenting.